predict_pv_yield/models/metrics.py

```diff
@@ -0,0 +1,25 @@
+import torch
+
+
+def mse_each_forecast_horizon(output: torch.Tensor, target: torch.Tensor) -> torch.Tensor:
+    """
+    Get MSE for each forecast horizon
+    output - The model estimate of size (batch_size, forecast_length)
+    target - The truth of size (batch_size, forecast_length)
+
+    Return a tensor of size (forecast_length)
+    """
+
+    return torch.sum((output - target) ** 2, dim=0)
```

Comment:
Sorry if I'm being dumb but should this be `torch.mean` instead of `torch.sum`?  :)

(And same for `mae_each_forecast_horizon`)

If it is meant to be `sum` instead of `mean` (because you weight the values elsewhere) then maybe the function should have a different name so people don't use this function thinking it computes the mean!?  e.g. `sum_abs_error_...`???

Reply:
Yea, you are totally right, it should have bee torch.mean